<?xml version="1.0" encoding="UTF-8" ?>
<robot name="mock_robot" xmlns:xacro="http://www.ros.org/wiki/xacro">

  <!-- The robot link description -->
  <link name="base_link">
      <visual>
      <geometry>
        <cylinder length="0.3" radius="0.2"/>
        <origin xyz="0 0 0.15"/>
      </geometry>
    </visual>
  </link>
</robot>
